intermatic-pe653-pool-control-system.groovy

```diff
@@ -746,47 +815,49 @@ def getVSP_SPEED_84 ()        { 20 }				// VSP Speed bit mask. 01x = VSP1, 08x =
 // Received a ManufacturerProprietary message. Pull the important details and update the UI controls
 
 private process84Event(byte [] payload) {
-	log("DEBUG", "+++++ process84Event payload: ${payload}")
+	log("DEBUG", 2, "+++++ process84Event payload: ${payload}")
```

Comment:
This is how I originally had the logs too, but found inconsistent meaning in the 'log level' as reported by Hubitat - they changed how the menu items came back too many times (from bitmap to scalar and back) for me to tolerate it. Probably they've stabilized it now.